File: tests/fixtures/java/imports.java
package fixtures.imports;

import java.util.List;
import java.util.*;
import static java.util.Collections.emptyList;
import static java.util.Collections.*;

class Imports {
    List<String> values() {
        emptyList();
        singletonList("value");
        Set.of("value");
        return List.of();
    }
}

File: tests/fixtures/scala/imports.scala
package fixtures.imports

import scala.collection.mutable
import scala.util.{Try, Success => Ok, Failure => _}
import helpers._
import scala.concurrent.{Future as Async, *}
import ordering.given
import ordering.{given Ordering, given Conversion, *}

object Imports:
  def run(): Unit =
    Ok(1)
    mutable.Map()
    Async.successful(1)
    unknownWildcard()
